src/sections/transaction/ReviewTransactionForm.utils.tsx

```diff
@@ -184,14 +192,35 @@ export const useTransactionValues = ({
       .gt(0)
   }
 
+  const acceptedFeePaymentAssetIds =
+    acceptedFeePaymentAssets
+      .filter((acceptedFeeAsset) => acceptedFeeAsset?.data?.accepted)
+      .map((acceptedFeeAsset) => acceptedFeeAsset?.data?.id) ?? []
+
+  let displayEvmFeePaymentValue
+  let evmAcceptedFeePaymentAssetIds: string[] = []
+
+  if (isEvmAccount(account?.address) && evmPaymentFee?.data) {
+    displayEvmFeePaymentValue = evmPaymentFee.data.shiftedBy(
+      -NATIVE_EVM_ASSET_DECIMALS,
+    )
+
+    evmAcceptedFeePaymentAssetIds = assets.all
```

Comment:
update to this please evmAcceptedFeePaymentAssetIds = [assets.getAsset(import.meta.env.VITE_EVM_NATIVE_ASSET_ID).id]